src/web/mjs/connectors/templates/CoreView.mjs

```diff
@@ -18,13 +19,23 @@ export default class CoreView extends Connector {
         // The query to retrieve the single manga title from inside of this.queryManga
         this.queryMangaTitle = 'h2.series-list-title';
 
+        this.queryMangaTitleFromURI = 'h1.series-header-title';
```

Comment:
Does this work for all websites based on CoreView or do some of them need to overwrite this?

Reply:
works for all the sites that don't already have it over written

Reply:
i tested all the connectors that use coreview